admin-link/locales/fr.json.liquid

```diff
@@ -1,3 +1,3 @@
 {
-  "text": "link-extension"
+  "name": "link-extension"
```

Comment:
Can we update these to a more realistic one? This will look super similar to the handle `admin-link` and isn't very helpful. How about `"Product details"`?